lib/dockerize_rails/docker_commands.rb

```diff
@@ -31,5 +31,23 @@ def self.build
       puts
       1
     end
+
+    def self.start
+      status = 0
+      status += DockerStart.start_mysql
+      puts "-----"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->